.java-version

```diff
@@ -0,0 +1 @@
+1.8
```

Comment:
Nice! I'm all for tracking versions in a file based approach.
Which toolchain uses `.java-version` files?

I'm personally more familiar with [sdkman](https://sdkman.io), which uses [`.sdkmanrc` files](https://sdkman.io/usage#env).
I'm open to any solution that makes managing the versions easier :smile:

Reply:
`jenv` from Homebrew IIRC